sdk/testutil/testlog.go

```diff
@@ -21,7 +21,7 @@ func testLogLevel() hclog.Level {
 	if level != hclog.NoLevel {
 		return level
 	}
-	return hclog.Warn
+	return hclog.Trace
```

Comment:
Did you intend to leave this in?

The only downside I see is if someone were to run tests with the `-v` flag then this would dump a ton of logs. Something like INFO seems more appropriate and then you can override the level with the TEST_LOG_LEVEL env var.

Reply:
Nope, missed reverting that before committing. It's reverted now.